tests/pull.bats

```diff
@@ -249,3 +249,20 @@ load helpers
   umount /tmp/buildah-test
   rm -rf /tmp/buildah-test
 }
+
+@test "pull-policy" {
+  mkdir ${TESTDIR}/buildahtest
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy bogus alpine
+
+  run_buildah 125 pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  run_buildah 125 inspect alpine
+
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json alpine
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy never alpine
+  run_buildah pull --signature-policy ${TESTSDIR}/policy.json --policy missing alpine
```

Comment:
How do we check if the image hasn't been pulled?

I think we need to push an image to a local registry, pull it, then push another one to overwrite. Then a `--policy missing` should not pull the new image which we can determine by checking the digests.

Reply:
We can check the output to see if this string "Getting image source signatures" is in it.  If it's not, then no pull.  If it is, then pull.  That probably ought to be added in here.

Reply:
In preparation for shortnames work: should this be `pull ... docker.io/library/alpine` (FQIN)?